db/raw/add-hotel-plugin.sql

```diff
@@ -0,0 +1,1576 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+--
+-- Name: sqitch; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA sqitch;
+
+
+ALTER SCHEMA sqitch OWNER TO postgres;
+
+--
+-- Name: SCHEMA sqitch; Type: COMMENT; Schema: -; Owner: postgres
+--
+
+COMMENT ON SCHEMA sqitch IS 'Sqitch database deployment metadata v1.0.';
+
+
+--
+-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: 
+--
+
+CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
+
+
+--
+-- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: 
+--
+
+COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
+
+
+SET search_path = public, pg_catalog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: admin_account; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE admin_account (
+    id uuid NOT NULL,
+    attendee_id uuid NOT NULL,
+    hashed character varying DEFAULT ''::character varying NOT NULL,
+    access character varying DEFAULT ''::character varying NOT NULL
+);
+
+
+ALTER TABLE public.admin_account OWNER TO rams_db;
+
+--
+-- Name: approved_email; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE approved_email (
+    id uuid NOT NULL,
+    subject character varying DEFAULT ''::character varying NOT NULL
+);
+
+
+ALTER TABLE public.approved_email OWNER TO rams_db;
+
+--
+-- Name: arbitrary_charge; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE arbitrary_charge (
+    id uuid NOT NULL,
+    amount integer NOT NULL,
+    what character varying DEFAULT ''::character varying NOT NULL,
+    "when" timestamp without time zone NOT NULL,
+    reg_station integer
+);
+
+
+ALTER TABLE public.arbitrary_charge OWNER TO rams_db;
+
```

Comment:
I'm guessing you made this migration BEFORE you merged the ``panels`` plugin stuff, since this shouldn't be here as it's no longer part of the core repo.

This kind of thing seems like a good reason to keep migrations event-specific, BTW.  If each plugin does its own migrations, then let's say I do the thing I'm planning on doing this weekend and rip out the ``tabletop`` stuff into its own plugin (since we've all agreed it doesn't really belong in the core ``uber`` plugin).  If I do that, and each feature plugin has its own migrations, then I'd need to update the migrations for ``tabletop`` and ``panels`` and ``hotel`` and ``uber``.  That's... probably not ideal.